enterprise/internal/campaigns/changeset_history.go

```diff
@@ -105,6 +105,8 @@ func computeHistory(ch *campaigns.Changeset, ce ChangesetEvents) (changesetHisto
 	}
 	pushStates(openedAt)
 
+	isDraft := currentExtState == campaigns.ChangesetExternalStateDraft
```

Comment:
How about moving that further up to the other `var ( )` declarations?

It might also be good to add a one-line comment to explain that `isDraft` is _not_ synonymous with `currentExtState == campaigns.ChangesetExternalStateDraft` because it keeps track of state over time (if that's correct)

Reply:
The other nitpick I have:

```suggestion
	wasDraft := currentExtState == campaigns.ChangesetExternalStateDraft
```

Might make it more clear that this var is tracking whether the changeset was/is a draft irrespective of the current external state (that can be closed/merged)

Reply:
Done.

I did not rename it to `wasDraft`, because on the external code host it still IS a draft, we just don't tell about that anymore because in our state logic, closed is stronger binding.